src/Strings/PerformStringShift.java

```diff
@@ -0,0 +1,146 @@
+/** QUESTION
```

Comment:
This file is not needed, delete it and push again

Reply:
This was the original file I commited. @sekhnar can you place your solution in this file?